invalidPropsals.ts

```diff
@@ -0,0 +1,122 @@
+import snapshot from '@snapshot-labs/snapshot.js';
+const dotenv = require('dotenv');
+dotenv.config();
+global['fetch'] = require('node-fetch');
+
+import db from './server/helpers/mysql';
+import { jsonParse } from './server/helpers/utils';
+import { spaces } from './server/helpers/spaces';
+import { archiveProposal, loadSpaces } from './server/helpers/adapters/mysql';
+
+const tableName = 'messages';
+const backuptableName = 'messages_backup';
+
+
+const init = async () => {
+    loadSpaces().then(async ensSpaces => {

```

Comment:
It's better to take spaces from the hub directly calling `https://hub.snapshot.page/api/spaces`, we would have a more consistent result, sometime loadSpaces fail for some space randomly